src/components/apps/admin/publicationRequests/RequestListing.js

```diff
@@ -0,0 +1,267 @@
+import React, { useState } from "react";
+import { useTranslation } from "i18n";
+import { useQuery, useMutation, queryCache } from "react-query";
+
+import { build, EmptyTable } from "@cyverse-de/ui-lib";
+
+import DETableHead from "components/utils/DETableHead";
+import TableLoading from "components/utils/TableLoading";
+import {
+    getAppPublicationRequests,
+    adminPublishApp,
+    APP_PUBLICATION_REQUESTS_QUERY_KEY,
+} from "serviceFacades/apps";
+import DEDialog from "components/utils/DEDialog";
+import ErrorTypographyWithDialog from "components/utils/error/ErrorTypographyWithDialog";
+
+import ids from "../../ids";
+import constants from "../../constants";
+
+import {
+    Button,
+    IconButton,
+    Paper,
+    TableContainer,
+    Table,
+    TableBody,
+    TableCell,
+    TableRow,
+    Tooltip,
+} from "@material-ui/core";
+import PublicIcon from "@material-ui/icons/Public";
+
+/**
+ *
+ * @author sriram
+ *
+ * A component that displays a list of app publication requests.
+ *
+ */
+
+const appColumnData = (t) => [
+    { name: t("name"), enableSorting: false, key: "name" },
+    {
+        name: t("integratorName").split(":")[0],
+        enableSorting: false,
+        key: "integrator_name",
+    },
+    {
+        name: t("integratorEmail").split(":")[0],
+        enableSorting: false,
+        key: "integrator_email",
+    },
+    {
+        name: "Tools Used",
+        enableSorting: false,
+        key: "tools",
+    },
+    {
+        name: "",
+        enableSorting: false,
+        key: "publish",
+    },
+];
+
+const toolColumnData = (t) => [
+    { name: t("name"), align: "left", enableSorting: false, id: "name" },
+    {
+        name: t("imageName"),
+        align: "left",
+        enableSorting: false,
+        id: "image_name",
+    },
+    { name: t("tag"), align: "left", enableSorting: false, id: "tag" },
+];
+function ToolsUsed(props) {
+    const { tools, appName, parentId } = props;
+    const [toolsDialogOpen, setToolsDialogOpen] = useState(false);
+    const { t } = useTranslation("tools");
+    return (
+        <React.Fragment>
+            <Button color="primary" onClick={() => setToolsDialogOpen(true)}>
+                {t("viewTools")}
+            </Button>
+            <DEDialog
+                id={build(parentId, ids.PUBLICATION_REQUESTS.TOOLS_USED_DIALOG)}
```

Comment:
Minor as always, but you can remove the `parentId` from here and the rest of the elements in the dialog